src/index.js

```diff
@@ -67,12 +68,41 @@ const createContext = (ctx) => {
   return ctx
 }
 
-const importDefault = async filepath => {
-  const module = await import(url.pathToFileURL(filepath).href)
-  return module.default
+/** @type {import('jiti').JITI | null} */
+let jiti = null
+
+const loader = async filepath => {
+  try {
+    const module = await import(url.pathToFileURL(filepath).href)
+    return module.default
+  } catch (err) {
+    /* c8 ignore start */
+    if (!TS_EXT_RE.test(filepath)) {
+      throw err
+    }
+    if (!jiti) {
+      try {
+        jiti = (await import('jiti')).default(__filename, {
+          interopDefault: true
+        })
+      } catch (err) {
+        if (
+          err.code === 'ERR_MODULE_NOT_FOUND' &&
```

Comment:
Hey another thing, I'm not completely sure if this `ERR_` is always there. I'm seeing people using both versions (with and without ERR_, before we had the without ERR_ version here, but while testing locally ERR_ thing seemed to work, might be something to do with node version, not sure). We can probably change this to `err.code.endsWith('MODULE_NOT_FOUND')` 👀

Reply:
Sure. I will do it manually (I am changing code style a little to sync it with my other projects).

Reply:
> might be something to do with node version

Ah, actually found out. The current version is also correct. It throws ERR_ prefixed code when doing `import()` and non-prefixed code when doing `require()` 🤦‍♂️